### PR TITLE
Result page

### PR DESCRIPTION
**Pull request by git-0r**
# Created result page screen for quiz app
quiz5

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,10 +6,116 @@
...
+                <circle cx="12" cy="10" r="3" />
+                <circle cx="12" cy="12" r="10" />
+            </svg>
+        </div>
+    </nav>
+
+    <main class="main-content">
+
+        <div class="result-container">
+            <p class="heading-xl">Result</p>
```
Comment (manish-GH) [inline index.html]: Consider using header tag

File: style.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
+body {
+  margin: 0;
+}
+
+.navbar-input {
+  width: 80%;
+  height: 30px;
```
Comment (manish-GH) [inline style.css]: use rem

**Review by manish-GH — COMMENTED** [review]
Give some spacing or some kind of separation between each question. Or you could make each component as a card.

Both the UI and the code are good. It can be merged after implementing the recommended changes

File: style.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+@media screen and (max-width: 600px) {
+  :root {
+    font-size: 66.25%;
+  }
+
+  .result-container {
+    width: 90%;
+    height: auto;
+  }
+}
```
Comment (Sahil25061999) [inline style.css]: use em for media query

**Review by Sahil25061999 — APPROVED** [review]
rest looks good. loved the minimalism of the ui . As suggested by manish add spacing between different questions,this will improve user experience.

File: result_page.html [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+    </nav>
+
+    <main class="main-content">
+
+        <div class="result-container">
+            <header class="heading-xl">Result</header>
+            <div class="d-flex flex-justify-center">
+                <p class="text-xl">Final Score: 40/50</p>
+            </div>
+            <div class="quiz-solution">
```
Comment (tarunsankhla) [inline result_page.html:39]: Add comments in code

File: result_page.css [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+.bg-white-full {
+  background-color: var(--clr-white-full);
+}
+
+.quiz-solution li {
+  list-style: upper-alpha;
+}
+
+@media screen and (max-width: 600px) {
```
Comment (tarunsankhla) [inline result_page.css:50]: Create a variable for media queries screen size

**Review by tarunsankhla — APPROVED** [review]
Overall you have used semantic html and good classnames, only minor changes

**Pull request merged**